examples/2d/plot_filters.py

```diff
@@ -54,29 +54,31 @@ def colorize(z):
     filter_c = np.fft.fftshift(filter_c)
     axs[i // L, i % L].imshow(colorize(filter_c))
     axs[i // L, i % L].axis('off')
-    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}".format(i // L, i % L))
+    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}$".format(i // L, i % L))
     i = i+1
 
-
-# Add blanks for pretty display
-for z in range(L):
-    axs[i // L, i % L].axis('off')
-    i = i+1
+fig.suptitle("Wavelets for each scales $j$ and angles $\\theta$ used" #, with the corresponding low-pass filter."
+                 "\n The contrast corresponds to the amplitude and the color to the phase.", fontsize=13)
```

Comment:
Color saturation and color hue respectively denote complex magnitude and complex phase.